packages/expo-clipboard/src/Clipboard.types.ts

```diff
@@ -0,0 +1,39 @@
+// @needsAudit
+export interface GetImageOptions {
+  /**
+   * The format of the clipboard image to be converted to.
+   */
+  format: 'png' | 'jpeg';
+  /**
+   * Specift the quality of the returned image, between 0 and 1. Defaults to `1` (highest quality).
+   * Applicable only when `format` is set to `jpeg`, ignored otherwise.
+   * @default 1
```

Comment:
```suggestion
   * Specify the quality of the returned image, between `0` and `1`. Defaults to `1` (highest quality).
   * Applicable only when `format` is set to `jpeg`, ignored otherwise.
   * @default 1
```

Luke's suggestion + formatting.